installers/metrics/ansible/roles/pgo-metrics/tasks/cleanup.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Cleanup Metrics Resources
+  tags:
+    - update-metrics
+    - uninstall-metrics
+  block:
+    - name: Delete Prometheus Deployment
+      command: "{{ kubectl_or_oc }} delete deployment crunchy-prometheus -n {{ metrics_namespace }}"
+      ignore_errors: yes
```

Comment:
For these `delete` calls, we could use `--ignore-not-found=true` and not ignore errors.

Reply:
The thought here is to continue on any error, not only "not found" errors.

Reply:
Adding `--ignore-not-found=true` could def avoid polluting our Ansible logs though ... def something to consider for the future (for metrics and operator installers).